test/manual/amp-lightbox-gallery-launch.amp.html

```diff
@@ -40,30 +40,31 @@ <h1>Lightbox Carousel Description Test</h1>
   <amp-img src="https://images.unsplash.com/photo-1503843778847-2b8bdce2ed3d"
```

Comment:
Sorry for putting these changes in this PR. They are unrelated. This is just a correction to the size / width of the `<amp-img>`s which were previously incorrect.

Reply:
lgtm :)